src/ui/src/composables/useLogger.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-console */
+
+/**
+ * A simple abstraction to use logger in the application. For the moment, it's just a proxy to `console`, but it can be plugged to any library later.
+ */
+export function useLogger(): Pick<
```

Comment:
What the interest of using a hook syntax here and not use a function as ?

```
logger.log("my logger")
```

instead of 

```
useLogger().log()
```